lib/tutorials/en_US/caching.md

```diff
@@ -70,13 +87,19 @@ return h.response(result).etag('xxxxxxxxx');
 
 Check the documentation of `etag` under the [response object](/api#response-object) for more details about the arguments and available options.
 
-### Server side caching
+## <a name="server-side" /> Server-side Caching
 
-hapi provides powerful, convenient server side caching via [catbox](https://www.github.com/hapijs/catbox). This tutorial section will help you understand how to use catbox.
+hapi provides powerful, convenient server side caching via [catbox](https://www.github.com/hapijs/catbox). This tutorial section will help you understand how to use `catbox`.
```

Comment:
Some places we've written "server-side" and other places "server side."  I'd propose we use the latter throughout the tutorial

Reply:
Yeah, that works.  Should I switch "client-side" with "client side"?

Reply:
Ah, sorry!  I meant the former.  I like "server-side" and "client-side."  I don't mind either style, though, as long as it's consistent :)